lib/backend/lite/lite.go

```diff
@@ -887,6 +800,92 @@ func (l *Backend) DeleteRange(ctx context.Context, startKey, endKey []byte) erro
 	})
 }
 
+func (l *Backend) ConditionalUpdate(ctx context.Context, i backend.Item) (*backend.Lease, error) {
+	if i.Key == nil {
+		return nil, trace.BadParameter("missing parameter key")
+	}
+
+	rev := backend.CreateRevision()
+	err := l.inTransaction(ctx, func(tx *sql.Tx) error {
+		now := l.clock.Now().UTC()
+		stmt, err := tx.PrepareContext(ctx, "UPDATE kv SET value = ?, expires = ?, modified = ?, revision = ? WHERE key = ? AND revision = ? AND (expires IS NULL OR expires > ?)")
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		defer stmt.Close()
+
+		result, err := stmt.ExecContext(ctx, i.Value, expires(i.Expires), id(now), rev, string(i.Key), i.Revision, now)
```

Comment:
I don't think we get anything out of a separate prepare and exec step, if done like this, so we can just `tx.ExecContext()` instead.